File: 2574. Left and Right Sum Differences/Solution.py
class Solution:
    def leftRigthDifference(self, nums: List[int]) -> List[int]:
        n = len(nums)
        prefix = [i for i in nums]
        for i in range(1, n):
            prefix[i] += prefix[i - 1]
        return [abs((prefix[i] - nums[i]) - (prefix[-1] - prefix[i])) for i in range(n)]

File: 2574. Left and Right Sum Differences/Solution.java
class Solution {
    public int[] leftRigthDifference(int[] nums) {
        int n = nums.length;
        int[] prefix = new int[n], ans = new int[n];
        prefix[0] = nums[0];
        for (int i = 1; i < n; i++) prefix[i] = nums[i] + prefix[i - 1];
        int right = prefix[n - 1];
        for (int i = 0; i < n; i++) {
            ans[i] = Math.abs((prefix[i] - nums[i]) - (right - prefix[i]));
        }
        return ans;
    }
}